### PR TITLE
feature/pdp-image-carousel-redesign --pdm-stg

### PR DESCRIPTION
**Pull request by montojdev**
# Task name - [QR - PDP Image Carousel Redesign AB test](https://app.clickup.com/t/86dvabwxm)

## Type of pull request
- [X] New Feature
- [ ] Bugfixes
- [ ] Release
- [ ] Cleanup
- [ ] Other (You must specify)

## Work Done
A new PDP section with sliders in order to unifed all the product images.

## Screenshots
image

## Preview link
https://www.qureskincare.com/?ig-preview=b7dc3d24-bf24-40ed-ad0d-2f4ea5bf6b1c&preview_theme_id=146953994479

### CODE REVIEW

File: snippets/pdm_before_after_carousel.liquid
Comment (samu3le) [inline snippets/pdm_before_after_carousel.liquid:1]: add comments to snippet

File: snippets/pdm_icons.liquid
Comment (samu3le) [inline snippets/pdm_icons.liquid:1]: add comments to snippet

File: snippets/pdm_product_carousel_unified_images.liquid
Comment (samu3le) [inline snippets/pdm_product_carousel_unified_images.liquid:1]: add comments to snippet

File: assets/endrock.js
Comment (samu3le) [inline assets/endrock.js:1]: add comments to function

**Review by samu3le — APPROVED** [review]

**Comment by montojdev** [comment]
comments added

**Pull request merged**